### DISCUSSION TITLE
Module not found: Can't resolve 'fs'

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#22913] [Help]
When I execute npm run dev, I can't get the 'fs' import to work. I get the error: Module not found: Can't resolve 'fs'. This seems to only be an error when I try to use fs-derived data in getStaticProps, as in [id].js. Can anyone help? ...I really don't want to do npm run eject and fiddle with the webpack.config file.

**Comment by stigkj** [comment]
This should work out of the box. Do you have a reproducible example?

**Reply by ghost** [reply]
never mind...my bad. It works.
